server/migrations/20230514095648-add-language-to-users.js

```diff
@@ -0,0 +1,29 @@
+exports.up = async function (db) {
+  await db.runSql(`
+    ALTER TABLE users
+    ADD COLUMN language text;
+  `)
+
+  // Set existing users to english language
+  await db.runSql(`
```

Comment:
No, existing users should have nothing set and they should be migrated when they log in, that was the whole idea behind this